frontend/system/src/Admin/EditDrug/helpers.ts

```diff
@@ -55,11 +56,78 @@ export const buildDrugInputFromEntity = (entity: EntityDetails): DrugInput => {
   };
 };
 
+export const buildEntityFromDrugInput = (
+  drug: DrugInput
+): UpsertEntityInput => {
+  return {
+    parentCode: '933f3f00', // Drug parent code
+    code: drug.code,
+    name: drug.name,
+    description: drug.name,
+    type: 'Product',
+    category: 'Drug',
```

Comment:
NIT for now, but wondering about defining these types and categories as enums on the backend that we could expose for the frontend? Save us some inevitable typo?

Reply:
Yeah I think this is a good idea. However, the backend doesn't know about this enums either (I don't think) they're defined in the loader at the moment and stored as strings... So maybe frontend enum/strings are just as good?

Reply:
Yeah that would be good too! 🚀

Reply:
https://github.com/msupply-foundation/unified-codes/pull/489/commits/5d5d07e210ab6cddb9f98174187045bc580b072d